pkgs/by-name/lu/lumafly/package.nix

```diff
@@ -17,18 +18,34 @@ buildDotnetModule rec {
   src = fetchFromGitHub {
     owner = "TheMulhima";
     repo = "lumafly";
-    rev = "v${version}";
+    rev = "refs/tags/v${version}";
     hash = "sha256-GVPMAwxbq9XlKjMKd9G5yUol42f+6lSyHukN7NMCVDA=";
   };
 
+  # Use .NET 9.0 since 7.0 is EOL
+  postPatch = ''
```

Comment:
instead of patching the csproj, maybe this would work?

```
dotnetFlags = [ "-p:TargetFramework=net9.0" ];
```

Reply:
Yep, that works for this package

Reply:
Done!